tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/service/websub/advanced/WebSubDummyTestCase.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.ballerinalang.test.service.websub.advanced;
+
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * This class is a dummy test to trigger before groups.
+ */
+@Test(groups = "websub-test") // not necessary, introduced as a workaround to get @BeforeGroup to run before other tests
```

Comment:
Cant we use @BeforeSuite ?

Reply:
Don't think it will work here, since we rely on the test group now to start all the relevant services, for a particular test set, by grouping them into a single ballerina module that is run for the set.

For example, the websub services need to be started only when we are about to run the websub tests (the websub test group) not before the entire suite.